docs/partial_source/design/building_blocks.rst

```diff
@@ -344,36 +344,43 @@ The compiler takes in any Ivy function, backend function, or composition, and re
-|.. code-block:: python                  |.. code-block:: python                   |.. code-block:: python                   |
-|                                        |                                         |                                         |
-| def pure_ivy(x):                       | def pure_torch(x):                      | def mix(x):                             |
-|     y = ivy.mean(x)                    |     y = torch.mean(x)                   |     y = ivy.mean(x)                     |
-|     z = ivy.sum(x)                     |     z = torch.sum(x)                    |     z = torch.sum(x)                    |
-|     f = ivy.var(y)                     |     f = torch.var(y)                    |     f = ivy.var(y)                      |
-|     k = ivy.cos(z)                     |     k = torch.cos(z)                    |     k = torch.cos(z)                    |
-|     m = ivy.sin(f)                     |     m = torch.sin(f)                    |     m = ivy.sin(f)                      |
-|     o = ivy.tan(y)                     |     o = torch.tan(y)                    |     o = torch.tan(y)                    |
-|     return ivy.concatenate(            |     return torch.cat(                   |     return ivy.concatenate(             |
-|         [k, m, o], -1)                 |         [k, m, o], -1)                  |         [k, m, o], -1)                  |
-|                                        |                                         |                                         |
-| # input                                | # input                                 | # input                                 |
-| x = ivy.array([[1., 2., 3.]])          | x = torch.tensor([[1., 2., 3.]])        | x = ivy.array([[1., 2., 3.]])           |
-|                                        |                                         |                                         |
-| # create graph                         | # create graph                          | # create graph                          |
-| graph = ivy.compile_graph(             | graph = ivy.compile_graph(              | graph = ivy.compile_graph(              |
-|     pure_ivy, x)                       |     pure_torch, x)                      |     mix, x)                             |
-|                                        |                                         |                                         |
-| # call graph                           | # call graph                            | # call graph                            |
-| ret = graph(x)                         | ret = graph(x)                          | ret = graph(x)                          |
-+----------------------------------------+-----------------------------------------+-----------------------------------------+
+The best part about the graph compiler is that it will convert code from ANY framework, including Ivy itself, into a computation graph with each node written in the
```

Comment:
This is not correct, the graph compiler does not convert code between frameworks. The graph compiler cannot convert TensorFlow code into PyTorch code, this is the transpiler which is different. The task is to show how the same Ivy code compiled to a different graph depending on the backend we are compiling to.